### PR TITLE
Added OPEN_SOURCE_EVENTS.md

### LINKED ISSUE #1769: Add a List to mention Open Source Events
## 🚀 Feature

Add a List to mention Open Source Events

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

## Motivation

Add a List to mention Open Source Programs. NeoAlgo has been a part of many Open Source Programs like: 

- HakinCodes Contributor's Hack 
- DSC NSEC's Winter of Code
- DigitalOcean's Hacktoberfest 
- DevScript's DevScript Winter of Code 

Please create an `OPEN_SOURCE_EVENTS.md` and mention them in a bit more detail. If you would like to get some stats about these individual events, drop me a message on Slack and I can help with that.

### PR DESCRIPTION
**Pull request by vismitap**
Fixed #1769

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes

### Description
Added OPEN_SOURCE_EVENTS.md

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.
image
image
image
image

### CODE REVIEW

File: OPEN_SOURCE_EVENTS.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+## 🎉 Open source programs we have been a part of 🎉
+Open source is a term that originally referred to open source software (OSS). Open source software is code that is designed to be publicly accessible—anyone can see, modify, and distribute the code as they see fit. There are a lot of open source programs held throughout the year to encourage everyone's contribution and build awesome projects. NeoAlgo is such an open source organization and has participated in quite a few events like the following.
+
+
+### ✨ HakinCodes Contributor's Hack ✨
+  
+  <a href="https://hakincodes.tech/ch20.html"><img src="https://i.imgur.com/P3g5Olx.png" style="width:200px; height:200px;"></img></a>
```
Comment (ankitaggarwal23) [inline OPEN_SOURCE_EVENTS.md]: - Change the Hakincodes logo. 



This is the correct one.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Rest looks good to merge.

File: OPEN_SOURCE_EVENTS.md (hunk 1 above)
Comment (vismitap) [reply OPEN_SOURCE_EVENTS.md]: Done!

**Review by ankitaggarwal23 — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Comment by vismitap** [comment]
Please add the level label to this! 😅

**Pull request merged**